bin/cli.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-const CssShortener = require('../index');
+const CssShortener = require('css-shortener');
```

Comment:
The relative path was wrong. It should be ../lib/index instead ../index
I don’t think requiring the package itself is the right solution here. If you think otherwise, please let me know!

Reply:
Yeah, makes sense since the cli package is not different from the main one. Just pushed another commit with the correct reference :)